socorro/siglists/prefix_signature_re.txt

```diff
@@ -19,8 +19,10 @@ CERT_.*
 CFRelease
 _chkstk
 CleanupPerAppKey
+ConditionVariableFallback::wait.*
```

Comment:
See [addition](https://github.com/mozilla-services/socorro/pull/3846/commits/61f7d24ca4a226035fd7326ddcadd2590c334bd4)

```python
>>> import re
>>> regex = re.compile('ConditionVariableFallback::wait\(.*\)')
>>> regex.findall('ConditionVariableFallback::wait(_RTL_CRITICAL_SECTION*, unsigned long)')
['ConditionVariableFallback::wait(_RTL_CRITICAL_SECTION*, unsigned long)']
>>> regex = re.compile('ConditionVariableFallback::wait(.*)')
>>> regex.findall('ConditionVariableFallback::wait(_RTL_CRITICAL_SECTION*, unsigned long)')
['(_RTL_CRITICAL_SECTION*, unsigned long)']
```

Perhaps it doesn't matter *what* it matches on (which is what the brackets are for) as long as it matches *anything*.